sql-mysql-postgresql-nosql.mdx

```diff
@@ -0,0 +1,216 @@
+FROM products;
+
+```
+
+This query selects the columns **`product_name`**, **`price`**, and **`description`** from the **`products`** table. It retrieves all the records (rows) in the table.
+
+If you want to take it a step further and filter the results based on specific criteria, you can use the WHERE clause in SQL. For example, to retrieve only the products with a price less than 100, you can modify the query to reflect the following:
+
+```sql
+
+SELECT product_name, price, description 
+FROM products 
+WHERE price < 100;
+
+```
+
+This query will return the product name, price, and description for all products whose price is less than 100.
+
+All in all, SQL is not tied to any specific database system but instead acts as a universal language that can be used with different database management systems (DBMS), such as MySQL and PostgreSQL (to name a few). This means that once you learn SQL, you can apply your knowledge to work with a variety of databases.
+
+The thing is - though learning SQL basics can be relatively straightforward, there are different flavors, [dialects](https://learnsql.com/blog/what-sql-dialect-to-learn/), and versions of SQL out there, and it can be easy to get lost in the sauce.
+
+## SQL vs. NoSQL
+
+SQL and NoSQL are two different approaches to data management, each with its own strengths and drawbacks. Like we mentioned above, SQL is not just any language; it's a specialized programming language designed specifically for managing and manipulating data in [relational database management systems (RDBMS)](https://www.techtarget.com/searchdatamanagement/definition/RDBMS-relational-database-management-system).
+
+While SQL is fantastic for structured data, [NoSQL](https://www.datastax.com/what-is/nosql) (short for “not only SQL” or “non-SQL”) takes a more flexible approach to data management. NoSQL is not a language itself, but instead refers to a category of database management systems that share common characteristics. NoSQL databases break away from the traditional SQL approach and offer an alternative way to handle data that comes in different flavors - from document-oriented, key-value, columnar, and graph databases. 
+
+NoSQL databases, like MongoDB, Redis, DynamoDB, and Cassandra, usually have their own special query languages or APIs tailored to their data models and capabilities.
+
+Since NoSQL is great for unstructured or semi-structured data, it can handle data like documents, key-value pairs, graphs, and time-series data. However, with this flexibility comes some trade-offs. NoSQL databases usually can’t provide the same level of transactional guarantees or complex querying capabilities as traditional SQL databases. 
+
+Let’s break things down a little more and look into the terms mentioned above as well as the good (and the bad) of a couple databases out there. **You can also [skip ahead](#summing-it-up) if you just want a quick summary.**
+
+![](/images/database-platforms-trend.png)
```

Comment:
This image feels a bit large and doesn't seem to necessarily fit or add context to the blog. I think we could either replace or just ditch an image here.

<img width="1260" alt="CleanShot 2023-07-24 at 12 07 13@2x" src="https://github.com/xataio/mdx-blog/assets/25181823/e4bdd6ef-8c79-4b8f-9b78-78fbb235f528">